### PR TITLE
refactor(*): optimizations and add prettier

### PR DESCRIPTION
**Pull request by null8626**
hi

### CODE REVIEW

File: src/ArgumentParser.js [hunk 1]
```diff
@@ -188,7 +181,7 @@ class ArgumentParser {
 		usage = this.resolveUsageParser(usage);
 
 		let braceOpen = usage.optional ? "[" : "<";
-		let braceClose = usage.optional ? "]" : ">";
+		let braceClose = braceOpen;
```
Comment (deniz-blue) [inline src/ArgumentParser.js]: this change would make outputs become something like `!add <a< <b<` instead of desired `!add <a> <b>`

**Comment by deniz-blue** [comment]
other than that one small issue, this is a pretty good PR!

**Comment by null8626** [comment]
thanks

**Review by deniz-blue — CHANGES_REQUESTED** [review]
a

File: src/ArgumentParser.js (hunk 1 above)
Comment (null8626) [reply src/ArgumentParser.js]: fixed

**Review by deniz-blue — CHANGES_REQUESTED** [review]
😐 `[`

**Comment by null8626** [comment]
fixed again

**Review by deniz-blue — APPROVED** [review]

**Pull request merged**